WordPress/gutenberg
# Remove empty and unused click-outside component

This pull request seeks to remove an unused and presumably accidental empty file `editor/components/click-outside/index.js` introduced in #412.

__Testing instructions:__

There should be no impact by these changes. Verify that `components/click-outside` is not a dependency of any existing file.
